lib/spack/spack/schema/modules.py

```diff
@@ -188,52 +188,3 @@
     "additionalProperties": False,
     "properties": properties,
 }
-
-
-# deprecated keys and their replacements
-old_to_new_key = {"exclude_implicits": "hide_implicits"}
-
-
-def update_keys(data, key_translations):
-    """Change blacklist/whitelist to exclude/include.
-
-    Arguments:
-        data (dict): data from a valid modules configuration.
-        key_translations (dict): A dictionary of keys to translate to
-            their respective values.
-
-    Return:
-        (bool) whether anything was changed in data
-    """
```

Comment:
Did we ever warn about these properties? Also, if we remove, we should probably remove the attributes from the schema above too.

Reply:
there was just one ;p

Reply:
Ooh, you mean they are in the scheme. Didn't see that.

Reply:
@alalazo do you wanna submit a PR that removes them? 0.20 release notes say they were removed already ;p

Reply:
Ok, will do it